src/clientmetricreport/RawMetricReport.ts

```diff
@@ -0,0 +1,7 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+export default class RawMetricReport {
```

Comment:
nit: Maybe we could name this as `RawRTCStatReport` to avoid the name conflict with `RawMetricReport` type in stat collector.

Reply:
Removed this class.